gravitee-node-vertx/src/main/java/io/gravitee/node/vertx/configuration/HttpServerConfiguration.java

```diff
@@ -654,6 +845,8 @@ public HttpServerConfiguration build() {
         environment.getProperty(prefix + "ssl.keystore.type", keyStoreType);
```

Comment:
What if I don't specify the `keystoreType` in my configuration ? I believe that the `keystoreType` will be valued to **`null`** instead of the default value (which was 'jks' if I'm not wrong).

We should make sure the default behavior remains the same as previously.

Reply:
No you're right. I was misled by the default configuration in the gravitee.yaml file (https://github.com/gravitee-io/gravitee-gateway/blob/master/gravitee-gateway-standalone/gravitee-gateway-standalone-distribution/src/main/resources/config/gravitee.yml#L32-L33)

So, it means the original behavior is kept untouched 👍 🙂